### PR TITLE
2177 api v3 columns

### PR DESCRIPTION
**Pull request by aviveiros11**
#### Any background context you want to provide?
The effort to move to API v3 - specifically columns
#### What's this PR do?
translates API v2 into v3 with auto schema helper 
#### How should this be manually tested?
Use the `Try It` functionality rendered by the API page on SEED for v3
#### What are the relevant tickets?
#2177

### CODE REVIEW

File: seed/views/v3/organizations.py [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+                {
+                    'status': 'success',
+                    'column_mappings_deleted_count': cm_count,
+                    'columns_deleted_count': c_count,
+                }
+            )
+        except Organization.DoesNotExist:
+            return JsonResponse({
+                'status': 'error',
+                'message': 'organization with with id {} does not exist'.format(organization_id)
```
[the remark this answers is not in the document]
Comment (aviveiros11) [reply seed/views/v3/organizations.py:82]: After trying to just use the pk in place of the org_id.  I cannot get around the error of `TypeError at /api/v3/organizations/5/columns/
columns() got multiple values for argument 'pk'`.  This however does not appear when grabbing the oranization_id, even though the values are the same.

File: seed/views/columns.py [hunk 2]
```diff
@@ -151,53 +149,6 @@ def retrieve(self, request, pk=None):
             'column': ColumnSerializer(c).data
         })
 
-    @ajax_request_class
```
[the remark this answers is not in the document]
Comment (aviveiros11) [reply seed/views/columns.py:154]: Ah yes, I see now.

**Comment by adrian-lara** [comment]
Per our discussion, I ended up with some code changes given my testing. If you test the post create and get (all) columns endpoints, and they look good, feel free to merge 👍

**Comment by aviveiros11** [comment]
Looks good to me!  Good call on making the org id an optional param 👍

**Pull request merged**